packages/logo/example/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "../logo.scss";
+
+interface Props {
+    negative: boolean;
+    kicked: boolean;
+}
+
+const Logo = ({ negative, kicked }: Props) => (
+    <svg
```

Comment:
Is there a way we can use logo-react so that we don't have to maintain 2 separate logos? Now after my update it is out of sync and this will probably happen in the future as well.

Reply:
That would make a circular dependency 😕